ouroboros-network/src/Ouroboros/Network/PeerSelection/Governor/Monitor.hs

```diff
@@ -88,7 +88,8 @@ targetPeers PeerSelectionActions{readPeerSelectionTargets}
             )
       -- We simply ignore target updates that are not "sane".
 
-      let -- We have to enforce the invariant that the number of root peers is
+      let inSensitiveState = requiresBootstrapPeers bootstrapPeersFlag ledgerStateJudgement
```

Comment:
we renamed sensitive to `reuiqresBoostrapPeers`, maybe use `usingBootstrapPeers` instead?